gnovm/stdlibs/std/native.gno

```diff
@@ -55,7 +55,10 @@ func EncodeBech32(prefix string, bz [20]byte) Address {
 func DecodeBech32(addr Address) (prefix string, bz [20]byte, ok bool) {
 	return decodeBech32(string(addr))
 }
-
+func ZeroAddress() Address {
+	return Address(zeroAddress())
+}
+func zeroAddress() string
```

Comment:
This should be a constant, not a function and especially not a natively bound function

Reply:
[#82c9597](https://github.com/gnolang/gno/pull/2401/commits/82c9597c4b1fe8e16261dfa9c687ddd04b7c7729)
gno std.ZeroAddress now is a constant

Reply:
@thinhnx-var 

Why do you need the zero address function?

Reply:
@zivkovicmilos 
From early when I start this PR, I need an Address type in gno so I followed other approach to get an `Address` type. Sorry for not checking what `Address` type is.
But my mainly purpose is, I want the `gnovm std.ZeroAddress` is exactly from `tm2/pkg/std.ZeroAddress`. But in latest commit, I made it const with definition in gno:
``const (
	ZeroAddress Address = "g100000000000000000000000000000000dnmcnx"
)``